daemons/gptp/common/ieee1588port.cpp

```diff
@@ -134,6 +145,21 @@ IEEE1588Port::IEEE1588Port
 
 	pdelay_count = 0;
 	sync_count = 0;
+	bzero(link_delay, 4);
```

Comment:
Agree that memset should be used. Consider if sizeof(link_delay) should be used rather than 4*sizeof(int).